dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -132,6 +132,21 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1296
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="reactor.blockhound" />
+    </Target>
+    <Source>
+      <Class name="io.netty.util.internal.Hidden" />
+    </Source>
+    <Reason>
+      If BlockHound library is present in the class path, it loads the Hidden class to enable
+      Netty-BlockHound integration that detects blocking calls. In this case, the missing classes
```

Comment:
delete "that detects blocking calls" since it's not relevant to the linkage error

Reply:
Removed